app/assets/v2/js/grants/fund.js

```diff
@@ -150,7 +198,7 @@ $(document).ready(function() {
 });
 
 var waitforData = function(callback) {
-  if ($('#signature').val() != '') {
+  if ($('#wait').val() == false) {
```

Comment:
could we use `if (!$('#wait').val()) {` OR `if ($('#wait').val() === false) {`

Reply:
Opted for solution two as I also needed to change the value to a string. My original solution worked because the loose equals allowed me to check a string vs. a boolean. JavaScript 🤷‍♂️